cvmfs/malloc_heap.cc

```diff
@@ -107,12 +107,7 @@ MallocHeap::MallocHeap(uint64_t capacity, CallbackPtr callback_ptr)
   assert((capacity_ % 8) == 0);
   heap_ = reinterpret_cast<unsigned char *>(sxmmap(capacity + 7));
   uintptr_t head = uintptr_t(heap_) % 8;
-  if (head > 0)
```

Comment:
The code changes here I don't quite understand.  The original intent was to ensure 8 byte alignment.  It allocates 7 bytes more and then cuts little pieces at the beginning and the end.  Actually it's probably unnecessary because `mmap()` returns something aligned on a page boundary but I didn't find a garantuee for it...